inclusive-design/idrc
# GH-465: Add CMS editor components (resolves #464, #465, #466)

## Description

1. Add three Netlify CMS editor components: YouTube embed, Image and Text, Image Position
2. As Eleventy shortcodes generated by these components are only parsed for the "body" field in the markdown data file. These components are only made available in editors for body fields.

Note that as discussed with Vera and David, the preview for these components in the CMS authoring interface is not implemented.

## Steps to test

1. Go to /admin page to author;
2. Click any collection that has a "body" field such as "Page"; Click a data entry and scroll to the "body" field;
3. Click the "+" icon on the editor menu. These three new features should be listed;
4.  Test each feature.

**Expected behavior:** 
These features should work as described in tickets.
